source/docs/03-metadata.xml

```diff
@@ -1677,6 +1677,53 @@
+                        <specDesc key="repository"/>
+                        <specDesc key="identifier"/>
+                        <specDesc key="history"/>
+                     </specList>
+                  </p>
+                  <p>The <gi scheme="MEI">repository</gi> element is intended to contain a description of the institution or individual currently holding the bibliographic item. Its content is either extensive prose or structured, and thus better machine readable, information. The <gi scheme="MEI">history</gi> element on the other hand is a container for additional non-bibliographic details regarding the physical location of an item. It may contain, among others, one of the elements <gi scheme="MEI">acquisition</gi>, <gi scheme="MEI">exhibHist</gi> or <gi scheme="MEI">provenance</gi> to describe any events that coincided with a change of location like exhibitions, change of custody or alike.</p>
+                  <p>The following example demonstrates how to structure more detailed information about a repository (including the use of <gi scheme="MEI">identifier</gi>):</p>
+                  <p>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample088.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+                  <p>
+                     <specList>
+                        <specDesc key="treatHist"/>
+                     </specList>
+                  </p>
+                  <p>Conservation activities are often necessary to ensure the long-term preservation and integrity of manuscripts or printed sources. These conservation activities might include bookbinding, restoration, paper chemistry and alike. In MEI the <gi scheme="MEI">treatHist</gi> element is intended to hold records of any treatment an item has undergone and may even specify details of the individual treatment process. <gi scheme="MEI">treatHist</gi> allows either to opt for text or more structured information when formulating the specific encoding. It is recommended to make use of <gi scheme="MEI">p</gi> elements when a text-centred encoding is favoured and to use the <gi scheme="MEI">eventList</gi> element for a more structured encoding. Like <gi scheme="MEI">exhibHist</gi>, <gi scheme="MEI">treatHist</gi> is conceptually bound to the item-level, however, as another child element to <gi scheme="MEI">history</gi>, <gi scheme="MEI">treatHist</gi> is allowed in several places. It is at the discretion of the encoder to decide where the information is most appropriate for the particular project or encoding purposes. As best practice, it is recommended that <gi scheme="MEI">treatHist</gi> is encoded as a child element of either <gi scheme="MEI">item</gi> or <gi scheme="MEI">physDesc</gi>.</p>
```

Comment:
```suggestion
                  <p>Conservation activities are often necessary to ensure the long-term preservation and integrity of manuscripts or printed sources. These conservation activities might include bookbinding, restoration, paper chemistry and alike. In MEI the <gi scheme="MEI">treatHist</gi> element is intended to hold records of any treatment an item has undergone and may even specify details of the individual treatment process. <gi scheme="MEI">treatHist</gi> allows either to opt for text or more structured information when formulating the specific encoding. It is recommended to make use of <gi scheme="MEI">p</gi> elements when a text-centred encoding is favoured and to use the <gi scheme="MEI">eventList</gi> element for a more structured encoding. Like <gi scheme="MEI">exhibHist</gi>, <gi scheme="MEI">treatHist</gi> is conceptually bound to the item-level. As an element it is thus not permitted at the work or expression level and only permitted at the manifestation level, if the manifestation comprises a manifestation singleton.</p>
```

Reply:
Because of other changes this suggestion was outdated. I had to fix it manually. Done at 28487dd779ba1793fcee888bb7d865a4fb944ad8